### DISCUSSION TITLE
Which report for Peptide quantification in DIA-NN?

### DISCUSSION THREAD

**Discussion by wangrui85** [vdemichev/DiaNN#970] [General]
Hello,

I am firstly the DIA raw data using DIA NN software for peptide quantification. During this - while scrolling through the main output report from DIANN - I wondered which report are to consider for peptide quantification. For example, what are the columns of precursors ('pr_matrix') that allow me to have high confidence quantification results. I have read paper use parameters (3 MS1 acquisitions per peak and the signal-to-noise threhod set to 2.5). But how about DIA-NN to result peptide intensity?

Thanks in advance for any help!

**Comment by vdemichev** [comment]
We recommend using the main report in all cases. The matrices are there for 'a quick look' or for processing in Excel only.
In the main report, normally one would use Precursor.Normalised. One can aggregate those values into peptide quantities by taking the maximum of them per Stripped.Sequence or Modified.Peptide in each run. This can be done with a simple R script or e.g. with the diann_matrix function of the diann R package.

Best,
Vadim

**Comment by wangrui85** [comment]
Dear Vadim, thanks so much for quick reply! 

Could I take solution as two ways as peptide quntification?
1) directly used precursor normalised;
or 2) take the maximum of stripped or modified peptides.
But my confuse is how about unmodified/modified peptides? It seems that their difference (charges/modifications) present in "Precursor.Id". For example, "AAAAAAALQAK" showed charge"+1" and "+2". I only need consider the maximum"10413400" for peptide quntities? If this, it's unable to discriminate phosphopeptide and unphosphopeptide in a case?
![Screenshot 2024-03-19 104321](https://github.com/vdemichev/DiaNN/assets/87349846/79ec47ce-4514-46ef-a4d5-8a6dda115a9d)

sincerely, really appreciate it.

Rui

**Reply by wangrui85** [reply]
another question is whether I should remove precursor (1 charge) firstly?

**Comment by vdemichev** [comment]
'Modified.Sequence' in the output refers to the sequence identifier of the peptide including all modifications, if any. That is it's a valid entry also for unmodified peptides.

**Comment by wangrui85** [comment]
Got it. thanks so much！ I directly took the pr_matrix.tsv with modified sequences.

Great software！

from 阿里邮箱 iPhone------------------------------------------------------------------
发件人：Vadim ***@***.***>
日　期：2024年03月23日 02:37:51
***@***.***>
***@***.***>; ***@***.***>
主　题：Re: [vdemichev/DiaNN] Which report for Peptide quantification in DIA-NN? (Discussion #970)

'Modified.Sequence' in the output refers to the sequence identifier of the peptide including all modifications, if any. That is it's a valid entry also for unmodified peptides.
